source/includes/steps-define-custom-user-data-import-export.yaml

```
@@ -66,9 +66,14 @@ content: |
          data.
      
      * - ``user_id_field``
-       - The name of a field that contains an application user's ID
-         value. All user documents in the collection must have this
-         field and no two documents may contain the same user ID.
+       - The name of the field that maps each document to a specific application
+         user. The field must have the same name in each document and contain
```

Not sure if my suggestion is better, but can a field be considered the same if the name is different? Same comment for both instances.

```suggestion
         user. The field must be present in every document and contain
```

---

It technically doesn't have to be in every document, just in documents that you actually want to map to users. Updated for clarity.